IDEOorg/steps-bot
# #162413570 Refactor hardcoded status and topics into a constant file

#### What does this PR do?
- Refactors all hardcoded status and topics strings into constants

#### Description of tasks to be completed
- Find all instances where values for status and topic are hardcoded
- Define the identified values in the `constants.js` file and replace the hardcoded ones with values from the `constants.js` file

#### How should this be manually tested?
- Clone this repo and checkout to a branch named `ch-move-hard-coded-constants-to-constant-file-162413570`.
- Explore the `src` directory and confirm that there is a file named `constants.js` with `STATUS` and `TOPICS` constants defined there.
- Confirm that no hardcode strings for `status` and `topic` in the other files in `src` directory

#### Any background context you want to provide?
Having constants defined in a single file makes it possible to update them when needed rather than having to locate all instances where they have been used to update them. This improves code maintainability.

#### What are the relevant pivotal tracker stories?
[Refactor hardcoded status and topics into a constant file](https://www.pivotaltracker.com/story/show/162413570)

#### Screenshots?
N/A

#### Questions
N/A
